tools/perf/dashboard_auth.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env vpython3
+#
+# [VPYTHON:BEGIN]
+# wheel: <
+#   name: "infra/python/wheels/google-auth-py2_py3"
+#   version: "version:1.2.1"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/pyasn1-py2_py3"
+#   version: "version:0.4.5"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/pyasn1_modules-py2_py3"
+#   version: "version:0.2.4"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/six"
+#   version: "version:1.10.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/cachetools-py2_py3"
+#   version: "version:2.0.1"
+# >
+# wheel: <
+#   name: "infra/python/wheels/rsa-py2_py3"
+#   version: "version:4.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/requests"
+#   version: "version:2.13.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/google-api-python-client-py2_py3"
+#   version: "version:1.6.2"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/httplib2-py2_py3"
+#   version: "version:0.12.1"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/oauth2client-py2_py3"
+#   version: "version:3.0.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/uritemplate-py2_py3"
+#   version: "version:3.0.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/google-auth-oauthlib-py2_py3"
+#   version: "version:0.3.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/requests-oauthlib-py2_py3"
+#   version: "version:1.2.0"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/oauthlib-py2_py3"
+#   version: "version:3.0.1"
+# >
+#
+# wheel: <
+#   name: "infra/python/wheels/google-auth-httplib2-py2_py3"
+#   version: "version:0.0.3"
+# >
+# [VPYTHON:END]
+#
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+"""A tool to perform authorization to https://brave-perf-dashboard.appspot.com/
```

Comment:
@diracdeltas @thypon would we need an official sec review for this thing, or maybe a quick sign-off is possible?
the dashboard itself and the authentication idea was reviewed at https://github.com/brave/security/issues/656

Reply:
i'll take a look